references/oidc-mock/apiproxy/policies/AM-SetConsentPage.xml

```diff
@@ -0,0 +1,157 @@
+            border: 1px solid #b9b9b9;
+            border-top: 1px solid #a0a0a0;
+            -moz-box-shadow: inset 0 1px 2px rgba(0,0,0,0.1);
+            -webkit-box-shadow: inset 0 1px 2px rgba(0,0,0,0.1);
+            box-shadow: inset 0 1px 2px rgba(0,0,0,0.1);
+        }
+        .login {
+            text-align: center;
+            font-size: 14px;
+            font-family: 'Arial', sans-serif;
+            font-weight: 700;
+            height: 36px;
+            padding: 0 8px;
+        }
+        .login-submit {
+            border: 0px;
+            color: #fff;
+            text-shadow: 0 1px rgba(0,0,0,0.1);
+            background-color: #4d90fe;
+        }
+        .login-submit:hover {
+            border: 0px;
+            text-shadow: 0 1px rgba(0,0,0,0.3);
+            background-color: #357ae8;
+        }
+        .login-card a {
+            text-decoration: none;
+            color: #666;
+            font-weight: 400;
+            text-align: center;
+            display: inline-block;
+            opacity: 0.6;
+            transition: opacity ease 0.5s;
+        }
+        .login-card a:hover {
+            opacity: 1;
+        }
+        .login-help {
+            width: 100%;
+            text-align: center;
+            font-size: 12px;
+        }
+        .login-client-image img {
+            margin-bottom: 20px;
+        }
+        .login-card input[type=checkbox] {
+            margin-bottom: 10px;
+        }
+        .login-card label {
+            color: #999;
+            font-size: 16px;
+        }
+        .grant-debug {
+            font-family: Fixed, monospace;
+            width: 100%;
+            text-align: center;
+            font-size: 12px;
+            color: #999;
+            padding-top: 10px;
+        }
+        button {
+            cursor: pointer;
+        }
+    </style>
+</head>
+<body>
+<div class="login-card">
+    <h1>Consent Page</h1>
+    <div class="login-client-image">
+       <img src="https://www.logo.wine/a/logo/Apigee/Apigee-Logo.wine.svg" width="50%">
+    </div>
+    <form autocomplete="off" action="/v1/openid-connect/confirm" method="post">
+        <div class="grant-debug">
+         client app: {oidc.flow.token.app_name} - scopes: {oidc.flow.token.scope}
+        </div>
+        <button type="submit" name="reject" id="reject" class="login">Reject</button>
+        <button type="submit" name="submit" id="submit" class="login login-submit">Allow</button>
+        <input required type="hidden" name="state" value="{oidc.flow.token.state}">
+        <input required type="hidden" name="redirect_uri" value="{oidc.flow.token.redirect_uri}">
+    </form>
+    <div class="login-help">
+        <a href="#">[ Cancel ]</a>
+        <a href="#">[ Terms of Service ]</a>
+        <a href="#">[ Privacy Policy ]</a>
+    </div>
+</div>
+</body>
+</html> ]]></Payload>
+    </Set>
+    <IgnoreUnresolvedVariables>true</IgnoreUnresolvedVariables>
```

Comment:
what do you want to happen if a `oidc.flow.*` is null? do you want it to be ignored?